app/assets/javascripts/backbone/views/media_view.coffee

```diff
@@ -1,20 +1,56 @@
-# Handles clicks on media objects and manages the media modal.
+# Handles events on all media browser elements on the page.
+# Controls the media modal (no more than one per page).
 class MS.Views.MediaView extends Backbone.View
 
+  el: 'body'
+
   events:
```

Comment:
This file should be cleaned up/ refactored. Some parts are reusable, but some are not. Function names may need to be adjusted for delete media actions to be clearer.